clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json

```diff
@@ -2,7 +2,7 @@
     {
         "schema_version": "1.6.0",
         "id": "GHSA-462w-v97r-4m45",
-        "modified": "2023-11-08T04:00:58.644982Z",
+        "modified": "1970-01-01T00:00:00Z",
```

Comment:
These don't look like dates from new actual results. Did you modify them manually?